diff -Naur a/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in b/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in
--- a/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in	2023-08-31 19:54:31.875957934 +0530
+++ b/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in	2023-08-31 19:57:23.820717117 +0530
@@ -3,7 +3,7 @@
 Name=Tagger
 Comment=Tag your music
 Exec=@EXEC@ %u
-Icon=org.nickvision.tagger
+Icon=snap.tagger.org.nickvision.tagger
 Terminal=false
 Type=Application
 Categories=AudioVideo;Audio;
diff -Naur a/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in b/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in
--- a/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in	2023-08-31 19:54:31.875957934 +0530
+++ b/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in	2023-08-31 19:57:42.663046973 +0530
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>org.nickvision.tagger</id>
+  <icon type="stock">snap.tagger.org.nickvision.tagger</icon>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>MIT</project_license>
   <name>Tagger</name>
