diff -Naur a/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in b/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in
--- a/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in	2023-06-29 12:23:33.044779414 +0530
+++ b/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in	2023-06-29 12:42:43.618784902 +0530
@@ -3,10 +3,10 @@
 Name=Tagger
 Comment=Tag your music
 Exec=@EXEC@
-Icon=org.nickvision.tagger
+Icon=snap.tagger.org.nickvision.tagger
 Terminal=false
 Type=Application
 Categories=AudioVideo;
 Keywords=Tag;Music;Editor;Library;
 X-GNOME-UsesNotifications=true
-StartupNotify=true
\ No newline at end of file
+StartupNotify=true
diff -Naur a/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in b/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in
--- a/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in	2023-06-29 12:23:33.044779414 +0530
+++ b/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in	2023-06-29 12:42:29.698989714 +0530
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>org.nickvision.tagger</id>
+  <icon type="stock">snap.tagger.org.nickvision.tagger</icon>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>MIT</project_license>
   <name>Tagger</name>
