diff -Naur a/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in b/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in
--- a/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in	2023-07-26 18:37:40.372473823 +0530
+++ b/NickvisionTagger.Shared/org.nickvision.tagger.desktop.in	2023-07-26 18:40:02.271944842 +0530
@@ -3,7 +3,7 @@
 Name=Tagger
 Comment=Tag your music
 Exec=@EXEC@
-Icon=org.nickvision.tagger
+Icon=snap.tagger.org.nickvision.tagger
 Terminal=false
 Type=Application
 Categories=AudioVideo;Audio;
diff -Naur a/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in b/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in
--- a/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in	2023-07-26 18:37:40.372473823 +0530
+++ b/NickvisionTagger.Shared/org.nickvision.tagger.metainfo.xml.in	2023-07-26 18:39:55.550646932 +0530
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>org.nickvision.tagger</id>
+  <icon type="stock">snap.tagger.org.nickvision.tagger</icon>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>MIT</project_license>
   <name>Tagger</name>
