diff --git a/NickvisionTagger.GNOME/Controls/CorruptedFilesDialog.cs b/NickvisionTagger.GNOME/Controls/CorruptedFilesDialog.cs
index 9af75e4..d008a4a 100644
--- a/NickvisionTagger.GNOME/Controls/CorruptedFilesDialog.cs
+++ b/NickvisionTagger.GNOME/Controls/CorruptedFilesDialog.cs
@@ -28,7 +28,7 @@ public partial class CorruptedFilesDialog : Adw.Window
         //Dialog Settings
         SetIconName(iconName);
         SetTransientFor(parent);
-        _helpButton.OnClicked += (sender, e) => Gtk.Functions.ShowUri(this, "help:tagger/corrupted", 0);
+        _helpButton.OnClicked += (sender, e) => Gtk.Functions.ShowUri(this, "https://htmlpreview.github.io/?https://raw.githubusercontent.com/NickvisionApps/Tagger/main/NickvisionTagger.Shared/Docs/html/C/corrupted.html", 0);
         foreach (var path in files)
         {
             var row = Adw.ActionRow.New();
diff --git a/NickvisionTagger.GNOME/Views/MainWindow.cs b/NickvisionTagger.GNOME/Views/MainWindow.cs
index 8384bd7..083b69f 100644
--- a/NickvisionTagger.GNOME/Views/MainWindow.cs
+++ b/NickvisionTagger.GNOME/Views/MainWindow.cs
@@ -352,7 +352,7 @@ public partial class MainWindow : Adw.ApplicationWindow
         application.SetAccelsForAction("win.quit", new string[] { "<Ctrl>q" });
         //Help Action
         var actHelp = Gio.SimpleAction.New("help", null);
-        actHelp.OnActivate += (sender, e) => Gtk.Functions.ShowUri(this, "help:tagger", 0);
+        actHelp.OnActivate += (sender, e) => Gtk.Functions.ShowUri(this, "https://htmlpreview.github.io/?https://github.com/NickvisionApps/Tagger/blob/main/NickvisionTagger.Shared/Docs/html/C/index.html", 0);
         AddAction(actHelp);
         application.SetAccelsForAction("win.help", new string[] { "F1" });
         //About Action
@@ -396,12 +396,12 @@ public partial class MainWindow : Adw.ApplicationWindow
         if (e.Action == "unsupported")
         {
             toast.SetButtonLabel(_("Help"));
-            toast.OnButtonClicked += (sender, e) => Gtk.Functions.ShowUri(this, "help:tagger/unsupported", 0);
+            toast.OnButtonClicked += (sender, e) => Gtk.Functions.ShowUri(this, "https://htmlpreview.github.io/?https://raw.githubusercontent.com/NickvisionApps/Tagger/main/NickvisionTagger.Shared/Docs/html/C/unsupported.html", 0);
         }
         else if (e.Action == "format")
         {
             toast.SetButtonLabel(_("Help"));
-            toast.OnButtonClicked += (sender, e) => Gtk.Functions.ShowUri(this, "help:tagger/format-strings", 0);
+            toast.OnButtonClicked += (sender, e) => Gtk.Functions.ShowUri(this, "https://htmlpreview.github.io/?https://raw.githubusercontent.com/NickvisionApps/Tagger/main/NickvisionTagger.Shared/Docs/html/C/format-strings.html", 0);
         }
         _toastOverlay.AddToast(toast);
     }
@@ -1172,7 +1172,7 @@ public partial class MainWindow : Adw.ApplicationWindow
     /// </summary>
     /// <param name="sender">Gtk.Button</param>
     /// <param name="e">EventArgs</param>
-    private void AdvancedSearchInfo(Gtk.Button sender, EventArgs e) => Gtk.Functions.ShowUri(this, "help:tagger/search", 0);
+                            private void AdvancedSearchInfo(Gtk.Button sender, EventArgs e) => Gtk.Functions.ShowUri(this, "https://htmlpreview.github.io/?https://raw.githubusercontent.com/NickvisionApps/Tagger/main/NickvisionTagger.Shared/Docs/html/C/search.html", 0);
 
     /// <summary>
     /// Occurs when the _listMusicFiles's selection is changed
